### DISCUSSION TITLE
Check if player has rights on claim? Plugin Development

### DISCUSSION THREAD

**Discussion by eloo-dev** [GriefPrevention/GriefPrevention#1018] [Q&A] [answered]
Hello i am programming a blockbreaker plugin and would like to check with support of GriefPrevention if the player who sets up the blockbreaker has the rights on the claim or you can only create the blockbreaker on one claim. can anyone tell me what the function is called or is there a developer documentation for griefprevention somewhere?

**Comment by RoboMWM** [comment, marked as answer]
DataStore#getClaimAt, think there's also allowBreak in the main instance too?

You could also consider trying a fake BlockBreakEvent too.
